static/src/javascripts-legacy/projects/common/modules/experiments/tests/contributions-epic-billionaire.js

```diff
@@ -0,0 +1,75 @@
+define([
+    'common/modules/commercial/contributions-utilities',
+    'common/utils/template',
+    'text!common/views/contributions-epic-equal-buttons.html'
+], function (
+    contributionsUtilities,
+    template,
+    contributionsEpicEqualButtons
+) {
+
+
+    return contributionsUtilities.makeABTest({
+        id: 'ContributionsEpicBillionaire',
+        campaignId: 'kr1_epic_billionaire',
+
+        start: '2017-02-09',
+        expiry: '2017-02-22',
+
+        author: 'Jonathan Rankin',
+        description: 'Tests a rewrite of the epic centered around our lack of a billionaire owner',
+        successMeasure: 'Conversion rate',
+        idealOutcome: 'Acquires many Supporters',
+
+        audienceCriteria: 'All',
+        audience: 0.2,
+        audienceOffset: 0.62,
+
+        variants: [
+            {
+                id: 'control',
+                maxViews: {
+                    days: 30,
+                    count: 4,
+                    minDaysBetweenViews: 0
+                },
+                template: function (contributionUrl, membershipUrl) {
+                    return template(contributionsEpicEqualButtons, {
+                        linkUrl1: membershipUrl,
+                        linkUrl2: contributionUrl,
+                        title: 'Since you’re here…',
```

Comment:
I checked the style guide and ellipses are supposed to have spaces either side in all instances